libnetwork/iptables/iptables.go

```diff
@@ -202,19 +202,6 @@ func (iptable IPTable) ProgramChain(c *ChainInfo, bridgeName string, hairpinMode
 		return errors.New("could not program chain, missing chain name")
 	}
 
-	// Either add or remove the interface from the firewalld zone
-	if firewalldRunning {
-		if enable {
-			if err := AddInterfaceFirewalld(bridgeName); err != nil {
-				return err
-			}
-		} else {
-			if err := DelInterfaceFirewalld(bridgeName); err != nil {
-				return err
-			}
-		}
-	}
```

Comment:
Can we safely remove this code here? Mostly wondering if there's code-paths we could hit where this logic is still needed. For example, I noticed there's a `n.registerIptCleanFunc()` used; https://github.com/moby/moby/blob/e0da5cb9291982a4c7b3c6b0642eeced4ba9f8ac/libnetwork/drivers/bridge/setup_ip_tables.go#L183-L185


Which I think would mean that these things are no longer cleaned up on `deleteNetwork()` 🤔

Reply:
I tried using DelInterfaceFirewalld inside deleteNetwork in the previous PR and it didn't work, not sure why...
Anyway, all the CI tests are good, as well as my tests.